### DISCUSSION TITLE
DEPRECATED CONSTANT

### DISCUSSION THREAD

**Discussion by oldchap56** [blakeblackshear/frigate#14882] [Config Support]
### Describe the problem you are having

MESSAGE IN HOME ASSISTANT 
MEDIA_CLASS_IMAGE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.IMAGE instead, please report it to the author of the 'frigate' custom integration

### Version

unknown

### Frigate config file

```yaml
df
```

### Relevant Frigate log output

```shell
df
```

### Relevant go2rtc log output

```shell
df
```

### Frigate stats

```json
df
```

### Operating system

HassOS

### Install method

HassOS Addon

### docker-compose file or Docker CLI command

```yaml
df
```

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

df

### Any other information that may be helpful

df

**Comment by hawkeye217** [comment]
This is already fixed and this is the wrong repo to report this in.
